docs/_static/_sphinx_javascript_frameworks_compat.js

```diff
@@ -0,0 +1,123 @@
+/* Compatability shim for jQuery and underscores.js.
+ *
+ * Copyright Sphinx contributors
+ * Released under the two clause BSD licence
+ */
+
+/**
+ * small helper function to urldecode strings
+ *
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#Decoding_query_parameters_from_a_URL
+ */
+jQuery.urldecode = function(x) {
+    if (!x) {
+        return x
+    }
+    return decodeURIComponent(x.replace(/\+/g, ' '));
+};
+
+/**
+ * small helper function to urlencode strings
+ */
+jQuery.urlencode = encodeURIComponent;
+
+/**
+ * This function returns the parsed url parameters of the
+ * current request. Multiple values per key are supported,
+ * it will always return arrays of strings for the value parts.
+ */
+jQuery.getQueryParameters = function(s) {
```

Comment:
```
jQuery.getQueryParameters = function(s = document.location.search) {
    const result = {};
    const parts = s.slice(s.indexOf('?') + 1).split('&');
    
    parts.forEach(part => {
        const [key, value] = part.split('=', 2).map(jQuery.urldecode);
        result[key] ? result[key].push(value) : result[key] = [value];
    });
    
    return result;
};

```

Reply:
I copied this Javascript file from somewhere and don't think it is worthwhile changing it (and we would need to change it in all repos for consistency).